Makefile

```diff
@@ -156,7 +159,7 @@ build/%.asc: ## Create release signatures for operator-sdk release binaries
 
 image: image-build image-push ## Build and push all images
 
-image-build: image-build-ansible image-build-helm image-build-scorecard-proxy image-build-scorecard-test ## Build all images
+image-build: image-build-ansible image-build-helm image-build-scorecard-proxy image-build-scorecard-test image-build-scorecard-test-kuttl## Build all images
```

Comment:
nit: should be a space before the `## Build all images`.

Reply:
thanks, will fix that in an upcoming PR to update the SHA image of the kuttl base image.